src/session/session_api.c

```diff
@@ -1682,6 +1682,28 @@ __session_query_timestamp(WT_SESSION *wt_session, char *hex_timestamp, const cha
     API_END_RET(session, ret);
 }
 
+/*
+ * __session_reset_snapshot --
+ *     WT_SESSION->reset_snapshot method.
+ */
+static int
+__session_reset_snapshot(WT_SESSION *wt_session)
+{
+    WT_DECL_RET;
+    WT_SESSION_IMPL *session;
+
+    session = (WT_SESSION_IMPL *)wt_session;
+    /* Return error if the isolation mode is read committed. */
+    if (session->txn->isolation == WT_ISO_READ_COMMITTED) {
+        WT_ERR_MSG(session, EPERM, "read committed isolation cannot reset snapshot.");
```

Comment:
It can be changed into `WT_RET_MSG` and also no need of `{}`

Reply:
Fixed